From ba9365bf669a497253f8f052666034eab36e6642 Mon Sep 17 00:00:00 2001
From: Cesar DE OLIVEIRA <cesarx.de.oliveira@intel.com>
Date: Wed, 30 Jan 2013 10:21:22 +0100
Subject: [PATCH 55/61] uevent: ttyIFXO node permission

BZ: 81140

ttyIFX0 is a dynamic node. Permission access should be configured with
uevent.

Change-Id: I89b05e0bf51bef7c625a0202697a66c0c7a8c8f0
Signed-off-by: Cesar DE OLIVEIRA <cesarx.de.oliveira@intel.com>
Reviewed-on: http://android.intel.com:8080/89361
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Marmounier, Edward <edward.marmounier@intel.com>
Reviewed-by: Berat, FredericX <fredericx.berat@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: Seibel, Eric <eric.seibel@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 rootdir/ueventd.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index ef4da15..0046d66 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -83,6 +83,7 @@
 /dev/ttyGS0               0660   root       radio
 /dev/mdmTrace             0660   root       radio
 /dev/mdm_ctrl             0660   root       radio
+/dev/ttyIFX0              0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

